barustenberg/src/srs/reference_string/file_reference_string.rs

```diff
@@ -52,8 +52,9 @@ impl FileReferenceString {
 
 impl ProverReferenceString for FileReferenceString {
     fn get_monomial_points(&self) -> Arc<Vec<G1Affine>> {
-        // Implementation depends on your project.
-        todo!()
+        // TODO: Is this what we want? Check over this cuz we may get away with an arc as a field
+        // in the struct instead.
+        Arc::new(self.pippenger.monomials())
```

Comment:
yeah put the arc (and evaluate if we also need an rwlock) inside please

Reply:
Fixed in first comment

Reply:
I'm not putting a RwLock since it looks like all accesses to the points are read only anyways.